README.md

```diff
@@ -33,6 +33,7 @@ The following table lists the configurable parameters of the Trino chart and the
 | `additionalConfigProperties` |  | `{}` |
 | `additionalLogProperties` |  | `{}` |
 | `additionalCatalogs` |  | `{}` |
+| `envVars` | Common Environment Variables applied to all Coordinator and Worker Pods | `[]` |
```

Comment:
How about naming it just `env` to match the Pod spec? We could also add `envFrom` but can do that in another PR.

https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core